javascript/app-wrangle/src/components/TableEditor.tsx

```diff
@@ -57,10 +57,14 @@ export const TableEditor: React.FC<TableEditorProps> = memo(
 		)
 
 		const tableName = useMemo(() => {
+			let name: string | null = null
 			const stepIndex = workflow.steps.findIndex(x => x.id === selectedTableId)
-			const name = upperFirst(workflow.steps[stepIndex]?.verb)
-			return stepIndex >= 0 ? `#${stepIndex + 1} ${name}` : selectedTableId
```

Comment:
We won't show the step number anymore?

Reply:
The logic is moved to the useTableName hook